### PR TITLE
fix: regex for capturing vercel preview links

### PR DESCRIPTION
**Pull request by yashimwong**
howdy?

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ inputs:
   preview_url_regexp:
     description: "Regular expression pattern to capture the preview URL. Ensure first capture group captures the preview URL."
     required: false
-    default: 'Preview: \[(.*)\]' # Do use single quotes, do NOT use delimiters.
+    default: '\[Visit Preview\]\((.*.sx)\)' # Do use single quotes, do NOT use delimiters.
```
Comment (aaimio) [inline action.yml:19]: Hello old friend! :wave: 

Could we make it backward compatible so we don't unintentionally break things? If you could patch up the regex I'll add some tests to it as well.

Thanks again! Hope you're doing well.

**Review by aaimio — CHANGES_REQUESTED** [review]

File: action.yml (hunk 1 above)
Comment (yashimwong) [reply action.yml]: Heyy @aaimio, I'm doing well (sort of) 😆 

 I accidentally added the regex check from work. Made it more generic. For the backwards compatibility I'll have to check if there's a way to check the Vercel bot version or something 🤔

Comment (aaimio) [reply action.yml]: No need, you could add a simple OR operator in the regex.

**Comment by batista** [comment]
hey guys, thanks for the project, whats the status of this? any chance this gets in? :)

**Comment by aaimio** [comment]
> hey guys, thanks for the project, whats the status of this? any chance this gets in? :)
> 
> BTW, you might want to make this matching lazy with `\[Visit Preview\]\((.*?)\)`, otherwise other data is being captured

Let me have a look later today and get this done @batista

**Comment by aaimio** [comment]
Fixed in https://github.com/aaimio/vercel-preview-url-action/pull/160

**Pull request closed without merging**